staff/durack/publication.html

```diff
@@ -643,10 +643,28 @@ <h2>Publications:</h2>
 <b>Data:</b><br>
 <br>
 
+<b>Durack, Paul J.</b>, Mark D. Zelinka, and Karl E. Taylor (2025) PCMDI AMIP ERSST 5.0 SST and sea-ice boundary
+conditions version 1.0 (CMIP7Plus). Version 20250827. <i>Earth System Grid Federation</i>.
+DOI: <a href="links/Duracketal25aESGF.html" target="_blank">10.25981/ESGF.input4MIPs.CMIP7Plus/2584105</a>
+<div data-badge-popover="right" data-badge-type="2" data-doi="10.25981/ESGF.input4MIPs.CMIP7Plus/2584105"
+data-hide-no-mentions="true" class="altmetric-embed altmetric-hidden"></div>
+<br>
+<b>Durack, Paul J.</b>, Mark D. Zelinka, and Karl E. Taylor (2025) PCMDI AMIP HadISST 1.1 SST and sea-ice boundary
+conditions version 1.0 (CMIP7Plus). Version 20250827. <i>Earth System Grid Federation</i>.
+DOI: <a href="links/Duracketal25bESGF.html" target="_blank">10.25981/ESGF.input4MIPs.CMIP7Plus/2584106</a>
+<div data-badge-popover="right" data-badge-type="2" data-doi="10.25981/ESGF.input4MIPs.CMIP7Plus/2584106"
+data-hide-no-mentions="true" class="altmetric-embed altmetric-hidden"></div>
+<br>
+<b>Durack, Paul J.</b>, Mark D. Zelinka, and Karl E. Taylor (2025) PCMDI AMIP OISST 2.1 SST and sea-ice boundary
+conditions version 1.0 (CMIP7Plus). Version 20250827. <i>Earth System Grid Federation</i>.
+DOI: <a href="links/Duracketal25cESGF.html" target="_blank">10.25981/ESGF.input4MIPs.CMIP7Plus/2584107</a>
+<div data-badge-popover="right" data-badge-type="2" data-doi="10.25981/ESGF.input4MIPs.CMIP7Plus/2584107"
+data-hide-no-mentions="true" class="altmetric-embed altmetric-hidden"></div>
+<br>
```

Comment:
I'm guessing these links will resolve in the future.  They are currently 404.

Reply:
Yeah these were all published yesterday, so it takes a day or so for the status to toggle and these to go live - so they are correct, but will resolve by next week